lib/fog/ovirt/requests/compute/attach_volume.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Ovirt
+      class Real
+        def attach_volume(id, options)
+          raise ArgumentError, "instance id is a required parameter" unless id
+          raise ArgumentError, "volume id is a required parameter for attach-volume" unless options.key? :id
+
+          client.attach_volume(id, options[:id])
```

Comment:
Nope. Keep the braces. It is a style change so nothing breaks.

The api is one to concern because all apis are like that the user would expect that to be the same so we can't change just this one. Also, this refactoring is out of the scope of this PR.

Feel free to create another PR to change this. =)